x-pack/filebeat/module/zeek/intel/_meta/fields.yml

```diff
@@ -0,0 +1,80 @@
+- name: intel
+  type: group
+  default_field: false
+  description: >
+    Fields exported by the Zeek Intel log.
+  fields:
+
+    - name: seen
+      type: group
+      fields:
+        - name: indicator
+          type: keyword
+          description: >
+            The intelligence indicator.
+
+        - name: indicator_type
+          type: keyword
+          description: >
+            The type of data the indicator represents.
+
+        - name: host
+          type: keyword
+          description: >
+            If the indicator type was Intel::ADDR, then this field will be present.
+
+        - name: conn
+          type: keyword
+          description: >
+            If the data was discovered within a connection, the connection record should go here to give context to the data.
+
+        - name: where
+          type: keyword
+          description: >
+            Where the data was discovered.
+
+        - name: node
+          type: keyword
+          description: >
+            The name of the node where the match was discovered.
+
+        - name: uid
+          type: keyword
+          description: >
+            If the data was discovered within a connection, the connection uid should go here to give context to the data. If the conn field is provided, this will be automatically filled out.
+
+        - name: f
```

Comment:
I changed this to `f` from `fa_file` based on https://docs.zeek.org/en/stable/scripts/base/frameworks/intel/main.zeek.html#type-Intel::Seen. I don't have a log sample to verify it though.

Reply:
```
{"ts":1574161961.329821,"uid":"Cm8HUc24IncCd8yW4c","id.orig_h":"172.16.160.22","id.orig_p":54919,"id.resp_h":"172.16.160.23","id.resp_p":445,"seen.indicator":"9052d06c6ac53471f8496263f8fef2eb","seen.indicator_type":"Intel::FILE_HASH","seen.where":"Files::IN_HASH","seen.node":"worker-2-4","matched":["Intel::FILE_HASH"],"sources":["Safe 32bit putty.exe"],"fuid":"FTsQFK277gbWqdNURh","file_mime_type":"application/x-dosexec","file_desc":""}

```

this is a row of mine, but  f ( f_a_file)  is not reported (also if I have https://docs.zeek.org/en/stable/scripts/base/frameworks/intel/files.zeek.html enabled)